### PR TITLE
Allow backend version to be displayed in the frontend

### PR DESCRIPTION
**Pull request by leanjunio**
On the backend:
- I created a route that is responsible for fetching `package.json().version` and allowed the route to respond that to get requests on `/api/v1/version`

On the frontend:
- I used the ApiService that was created in order to create a GET request from the App component
- Since the request returns a promise, I made sure to put an appropriate string inside the `version` variable that supposedly holds the version that is requested from the backend
- Also added some some CS to allow `backend version` to be output on the bottom right corner

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -27,7 +30,52 @@
...
+    apiService
+      .get(`/api/v1/version`)
+      .then(res => (this.version = res.data))
+      .catch(({ request }) => {
+        this.version = "not found";
+        if (request.status === 404) {
+          this.errorMessage = "No version found.";
+        } else {
+          this.errorMessage = "Failed to fetch results.";
+        }
```
Comment (monarchwadia) [inline frontend/src/App.vue:77]: Set `this.version = "Unknown"`

File: frontend/src/App.vue [hunk 2]
```diff
@@ -27,7 +30,52 @@
...
+      });
```
Comment (monarchwadia) [inline frontend/src/App.vue:78]:
```suggestion
      .catch(({ request }) => {
        this.version = "Unknown";
      });
```

File: backend/index.js [hunk 3]
```diff
@@ -36,6 +36,12 @@ async function start () {
     }
   });
 
+  server.route({
+    method: 'GET',
+    path: '/api/v1/version',
+    handler: (request, h) => version
```
[the remark this answers is not in the document]
Comment (leanjunio) [reply backend/index.js]: :+1:

File: frontend/src/App.vue [hunk 4]
```diff
@@ -11,6 +11,9 @@
     </div>
     <div class="accent-bar top"/>
     <div class="accent-bar bottom"/>
+    <div class="version">
+      <h6>Backend Version: {{version}}</h6>
```
[the remark this answers is not in the document]
Comment (leanjunio) [reply frontend/src/App.vue]: :+1:

**Comment by leanjunio** [comment]
Awesome, thanks for the feedback! I've made the changes you've requested.

**Comment by leanjunio** [comment]
Actually, please don't merge yet, just found a bug with the frontend. The versions stay at the bottom corner of the window not the App component exactly.
image

**Comment by monarchwadia** [comment]
Thanks for the work, @leanjunio ! Fixed and merged!

**Pull request merged**